app/views/devise/confirmations/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<h2>Resend confirmation instructions</h2>
+<h2><%= t('.resend_confirmation_instructions') %></h2>
```

Comment:
Add a space after `<h2>` and before `</h2>`

Reply:
this views are auto-generated by devise